src/services/dumpers/agent-nodejs-dumper.ts

```diff
@@ -0,0 +1,176 @@
+import type { ConfigInterface, DbConfigInterface } from '../../interfaces/project-create-interface';
+
+import _ from 'lodash';
```

Comment:
No library imports, this should be added in a utils (see utils/strings) and added in context.

Reply:
will do